docs(fix): wording MTA-1561 (#202)
scaleway/docs-content: dedibox-network/rpn/concepts.mdx

## RPN VPN

RPN VPN allows you to connect to your RPN group using a VPN. The VPN is based on OpenVPN to ensure easy configuration. You can add the VPN server to your RPN group as any other server.

> Notably, it allows you to securely connect from anywhere in your Private Network as well as to connect servers which don't have a RPN interface to the RPN.

Reviewer: I think here it isn't a product name (only for Elements)

Author: ```suggestion
Notably, it allows you to securely connect from anywhere in your private network as well as to connect servers that do not have a RPN interface to the RPN.
```
